EjercicioDiaTres.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjerciciosProyect_Visual.Net
+{
+    class EjercicioDiaTres
+    {
+        static EjercicioDiaTres ejercicioDia;
+        public static void Main(string[] args)
+        {
+            ejercicioDia = new EjercicioDiaTres();
+            ejercicioDia.llamarEjercicioUno();
+            ejercicioDia.llamarEjercicioDos();
+            ejercicioDia.llamarEjercicioTres();
+        }
+        public void llamarEjercicioUno()
+        {
+            int resultado;
+            Console.WriteLine("Ingresa el Primer número: ");
+            int numero1 = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingresa el Segundo número: ");
+            int numero2 = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingresa el Tercer número: ");
+            int numero3 = int.Parse(Console.ReadLine());
+            resultado = ejercicioDia.CalcularNumeroMayor(numero1, numero2, numero3);
+            Console.WriteLine("El Número Mayor de ellos es: " + resultado);
+            Console.ReadKey();
+        }
+        public void llamarEjercicioDos()
+        {
+            string nombredelmes;
+            Console.WriteLine("Digita un número del 1 al 12: ");
+            int numero = Convert.ToInt32(Console.ReadLine());
+            nombredelmes = ejercicioDia.DeterminarElMes(numero);
+            Console.WriteLine("El Número Digitado " + numero + " corresponde al Mes de: " + nombredelmes);
+            Console.ReadKey();
+        }
+        public void llamarEjercicioTres()
+        {
+            string numerosimpares;
+            Console.WriteLine("Ingrese un Número Positivo: ");
+            int numero = Convert.ToInt32(Console.ReadLine());
+            numerosimpares = ejercicioDia.MostrarNumerosImpares(numero);
+            Console.WriteLine("El Orden descendente al número " + numero + " es: " + numerosimpares);
+            Console.ReadKey();
+        }
+        public int CalcularNumeroMayor(int numero1, int numero2, int numero3)
```

Comment:
typo de las variables

```suggestion
        public int CalcularNumeroMayor(int numeroUno, int numeroDos, int numeroTres)
```

Reply:
Hecho